salt/tests/unit/modules/files/test_metalk8s_cordon.yaml

```diff
@@ -0,0 +1,52 @@
+CORDON_OBJECT_RESULT:
```

Comment:
Detail: for consistency, please write these lowercased as its already done for other files.